<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="searchTextEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Suchen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="dbView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="newButton">
        <property name="text">
         <string>Neuer Eintrag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plzButton">
        <property name="text">
         <string>Postleitzahlen Verwaltung</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKontakte">
    <property name="title">
     <string>Kontakte</string>
    </property>
    <addaction name="actionNeuer_Kontakt"/>
    <addaction name="actionVerlassen"/>
   </widget>
   <widget class="QMenu" name="menuPostleitzahlen">
    <property name="title">
     <string>Postleitzahlen</string>
    </property>
    <addaction name="actionVerwaltung_Postleitzahlen"/>
   </widget>
   <addaction name="menuKontakte"/>
   <addaction name="menuPostleitzahlen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNeuer_Kontakt">
   <property name="text">
    <string>Neuer Kontakt</string>
   </property>
  </action>
  <action name="actionVerlassen">
   <property name="text">
    <string>Verlassen</string>
   </property>
  </action>
  <action name="actionVerwaltung_Postleitzahlen">
   <property name="text">
    <string>Verwaltung Postleitzahlen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
